.github/workflows/pr.yml

```diff
@@ -12,15 +12,21 @@ jobs:
   lint-test:
     name: Validate code and charts
     runs-on: ubuntu-18.04
+    container: 
+      image: anndutova/csi-baremetal-devkit:latest
```

Comment:
To access private image can be used [container.credentials](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainercredentials)
```
      image: ghcr.io/dell/csi-baremetal/csi-baremetal-devkit:latest
      credentials:
        username: ${{ github.actor }}
        password: ${{ secrets.GITHUB_TOKEN }}
```